src/main.cpp

```diff
@@ -2247,6 +2277,12 @@ void vcRenderWindow(vcState *pProgramState)
     vcCamera_SwapMapMode(pProgramState);
 #endif
 
+  if (ImGui::IsKeyPressed(SDL_SCANCODE_F8))
```

Comment:
PrintScreen?

Reply:
Printscreen was being intercepted by the system/not working, might be to do with this keyboard's prntscreen being fn+ insert

Reply:
Printscreen seems to get taken up by the OS, at least on this keyboard